easybuild/easyconfigs/o/openjpeg/openjpeg-2.3.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,48 @@
+#
+# this file is for building the openjpeg library in Easybuild
+# Author: Fenglai Liu  (fenglai.liu@vanderbilt.edu)
+# Vanderbilt University
+#
+
+easyblock = 'CMakeMake'
+
+name = 'openjpeg'
```

Comment:
@murfreesboro Since the official software name is `OpenJPEG`, please use that (so you'll need to move this easyconfig to park it next to the existing `OpenJPEG-2.3.0-GCCcore-7.3.0.eb`)

Reply:
I renamed the eb file and remove it to the standard OpenJPEG folder.